docs/node/task-queues.md

```diff
@@ -0,0 +1,58 @@
+---
+id: task-queues
+title: Task Queues in Node
+sidebar_label: Task Queues
+---
+
+## What is a Task Queue?
+
+import SharedTaskQueuesBasic from '../shared/task-queues-basic.md'
+
+<SharedTaskQueuesBasic
+workflowLink="/docs/java/workflows"
+workerLink="/docs/java/workers"
+/>
+
+## How to use Task Queues
+
+In Node, a Task Queue is represented in code by name, as a `string`.
+There are 3 places where the name of the Task Queue is supplied by the developer.
+
+1. When starting a Workflow, you must pass the `taskQueue` option to the `Connection#workflow()` function.
```

Comment:
Please link to the API reference for `Connection.workflow`.
Why did you use the `#` sign? is it to mark that this is an instance method?

Reply:
This is the first use of `#` in the node docs, I don't mind using it as long as we're consistent.

Reply:
i personally think this is not understood by all js developers and may be confused with private class fields? but i'm not going to make a huge deal out of this really. linking the reference would be nice to have